kokoro/run_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o xtrace
+set -o pipefail
+
+export CI=kokoro
+
+export RUST_BACKTRACE=1
+export RUST_LOG=debug
+export XDG_RUNTIME_DIR=/var/run
+
+./scripts/docker_pull
+./scripts/docker_run nix develop .#ci --command just kokoro_run_tests
+
+mkdir -p "$KOKORO_ARTIFACTS_DIR/test_logs/"
+cp ./target/nextest/default/*.xml "$KOKORO_ARTIFACTS_DIR/test_logs/"
```

Comment:
```suggestion
cp ./target/nextest/default/*.xml "$KOKORO_ARTIFACTS_DIR/test_logs/" || true
```

Based on the other files.

Reply:
Done.